modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * etting-started-with-openshift-gitops.adoc
+
+[id="run-argo-cd-instance-on-cluster_{context}"]
+
+= Running the Argo CD instance at the cluster-level
+
+The default Argo CD instance installed by the OpenShift GitOps Operator as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
+. Label the existing nodes:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
++
+. Optional: If required, you can also apply taints and isolate the workloads on infrastructure nodes and prevent other workloads from scheduling on these nodes:
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add the `runOnInfra` toggle in the `GitOpsService` custom resource:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Optional: If taints have been added to the nodes, then add `tolerations` to the `GitOpsService` custom resource if taints have been added to the nodes, for example:
```

Comment:
custom resource, for example: 
Remove the extra 'if taints have been added to the nodes'